packages/doctype-tile/package.json

```diff
@@ -25,7 +25,7 @@
     "clean": "rm -rf ./lib"
   },
   "dependencies": {
-    "3id-blockchain-utils": "^1.0.0",
+    "3id-blockchain-utils": "^1.1.0",
```

Comment:
Is this package really needed in the tile doctype? cc @simonovic86

Reply:
it's not needed here, we can remove it 👍